rules-reviewed/hibernate/hibernate.windup.xml

```diff
@@ -22,7 +22,7 @@
             <javaclass references="org.hibernate.annotations.Type" matchesSource="{*}text{*}" />
          </when>
          <perform>
-            <hint title="@Type(type=text) is deprecated in Hibernate 4" effort="1" category-id="mandatory">
+            <hint title="Hibernate 4 - Deprecated @Type(type=text)" effort="1" category-id="optional">
```

Comment:
I disagree. This is not only about shifting the last word at the beginning. It is about consistency, conciseness, and simplicity of the titles. Without this reformating, the reports look like a mess.